### DISCUSSION TITLE
One-time donations

### DISCUSSION THREAD

**Discussion by Kubrix** [MonoGame/MonoGame#8260] [MonoGame Foundation]
I dont have Patreon so can i do a one time donation like these people https://monogame.net/donate/ ?

**Comment by tomspilman** [comment]
We started setting up a one time donation feature, but didn't finish.

We are considering a PayPal one time payment.  But also maybe thru GitHub Sponsors.

End of the day it should be something easy for us to manage and has low fees (a problem with Patreon IMO).

**Reply by BugInfestedCode** [reply]
Hi there,
I have recently come back to MonoGame after a long hiatus away from coding of any sort and I am interested in making a donation via PayPal. I don't want to have to set up an account elsewhere to donate. Thanks for all your hard work in making MonoGame what it is today.

**Reply by tomspilman** [reply]
@BugInfestedCode Thanks for reminding us!

https://www.paypal.com/donate/?hosted_button_id=K3K9QACYMZMUE

We will be updating the website to include this too soon.

**Reply by BugInfestedCode** [reply]
Thanks Tom, I will donate in the near future.

**Comment by BugInfestedCode** [comment]
Donated - Transaction ID: 8BS43087LH8926421

**Comment by Kubrix** [comment]
Why no https://ko-fi.com/ or https://start.easystaff.io/
NOt everyone have paypal
